src/etc/rc.gateway_alarm

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+#
```

Comment:
this file is better off with a copyright block and your copyright line

Reply:
It's a rip of the pf* file, I cannot really change it to copyright opnsense... :)

Reply:
it's a problem, because if they don't assert a license this is not open source licensed code tainting their product

then again, this isn't even called anywhere?

Reply:
line 1421 in gwlib.

Reply:
ok, here's a clean version based on our code. you can use it as is: /usr/local/etc/rc.monitor

Reply:
(https://github.com/opnsense/core/commit/6fb26f989)

Reply:
I could, but I would lose the alarm logging info wouldn't I?

Reply:
refactored what we have in apinger to prepare for your required changes so you plug the logging into it